### PR TITLE
Lesson bayes

### PR DESCRIPTION
**Pull request by canyon289**

### CODE REVIEW

**Comment by AlexAndorra** [comment]
The flow looks good to me @canyon289

File: lesson_bayes/Lesson_Bayes - Bayes Rule Intro.ipynb [hunk 1]
```diff
@@ -0,0 +1,198 @@
+{
```
Comment (twiecki) [inline lesson_bayes/Lesson_Bayes - Bayes Rule Intro.ipynb]: <p>expertment</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AlexAndorra/notebooks-experiments/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Comment (twiecki) [inline lesson_bayes/Lesson_Bayes - Bayes Rule Intro.ipynb]: <p>represents, I think there are spell checkers for JupyterNB.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AlexAndorra/notebooks-experiments/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

**Comment by twiecki** [comment]
Yeah, I think this looks good from a high level. Really showing how the posterior is computed for a single point using scipy is going to be instructive I think.

**Comment by canyon289** [comment]
Perfect, I'll get to it.

Thanks for the review

**Pull request closed without merging**